src/global/types.h

```diff
@@ -958,6 +958,8 @@ typedef enum GAMEFLOW_LEVEL_TYPE {
     GFL_RESTART = 7,
     GFL_SELECT = 8,
     GFL_BONUS = 9,
+    GFL_TITLE_DEMO_PC = 10,
```

Comment:
I'm not sure if there are more places that check for `GFL_TITLE`. In that case rather than sprinkling the codebase with additional `|| level_type == GFL_TITLE_DEMO_PC` across, I'd lean towards resetting the level type in the gameflow loader back to `GFL_TITLE` for the given level. Same for regular levels.